responsibleai/responsibleai/databalanceanalysis/aggregate_balance_measures.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+from typing import Callable, Dict, List
+
+import numpy as np
+import pandas as pd
+
+from responsibleai.databalanceanalysis.balance_measures import BalanceMeasures
+from responsibleai.databalanceanalysis.constants import Measures
+
+
+class _MeasureHelper:
```

Comment:
nit: I think it makes more sense to just have the methods (then you don't need to make them static) without the class